### ISSUE TITLE
Excuse me, about the reproducing of Pedstrian and Van on Kitti.

### ISSUE THREAD

**Issue by BryanChen024** [Ghostish/Open3DSOT#49]
Dear authors,

Great work and paper!
I am a postgraduate student and my research project is 3D SOT.
Regarding the result of reproducing on the KITTI dataset, I found that the results I reproduced on the Car and Cyclist data sets were basically the same, but the results on Pedestrian and Van were very different. 

The reproducing success/precision on Pedestrians are near **50/75**, much lower than the results in the paper **60/89**.

I just change the dataset name in the yaml cfg file (Car to Pedestrian) and then train it. I would be appreciated if you can share me more details on the model parameter setting if i fogot to set.

Thank you very much!

**Comment by Ghostish** [comment]
Can you reproduce the performance when using the provided checkpoint in this repo? If not, there may be something wrong with your data.

**Comment by BryanChen024** [comment]
> Can you reproduce the performance when using the provided checkpoint in this repo? If not, there may be something wrong with your data.

Yes, i can reproduce the performance with the pretrained model, but if i train the model ( change the dataset name in the yaml cfg file, from car to pedestrian), my test result will be low ┭┮﹏┭┮

**Comment by BryanChen024** [comment]
> Can you reproduce the performance when using the provided checkpoint in this repo? If not, there may be something wrong with your data.

So i just want to know if i have to change anything else except the category_name:

**Comment by BryanChen024** [comment]
> > Can you reproduce the performance when using the provided checkpoint in this repo? If not, there may be something wrong with your data.
> 
> So i just want to know if i have to change anything else except the category_name:
![image](https://github.com/Ghostish/Open3DSOT/assets/115997817/8c30c751-8878-4a59-bd54-91b1e431777e)

**Comment by BryanChen024** [comment]
And the car & cyclist 's results will be similar to the results of the paper, but Pedestrian and Van are much lower ┭┮﹏┭┮

**Comment by Ghostish** [comment]
Just try to use a smaller batch size. Please refer to #33.

**Comment by BryanChen024** [comment]
Thanks very much, i have already seen #33  and i found the preload data was 1.3GB.
i will try to type a smaller bs when start the training.
